js/model.js

```diff
@@ -1,5 +1,7 @@
 'use strict';
 
+const enableHelicopter = false; //This is for totally hide helicopter settings from user until all heli features will be ready
```

Comment:
So if I understand correctly, this is a dead code for now?
I think I saw a FEATURE for helis added. Maybe after all instead of false we can have it set to true if feature is set? 
After all, it's simpler to disable a feature than disable the feature and some variables in different tabs

Reply:
Nope, we don't have a feature for HELIs. I think we should expose board capabilities in some MSP messages so Configurator can hide features that board can't support.

Reply:
I think all FCs will work to some extend with this. If FC code will be merged of course. 
I suggest to assume this to true, drop the constant and only merge when FC will be able to handle those new settings

Reply:
I don't know how fast code changes in this repo. So I desided to split my work on at least two parts for code in my branch doesn't become too old and make conflicts. I need a couple of weeks more to complete this to flyable state. This constant just hides what I created, because if this code will be merged the unready helicopter features must not be available for users.

Reply:
50 is the default value for textfield wasn't empty when opened